ci/dev/vscode.patch

```diff
@@ -3098,6 +3105,20 @@ index ba2701ec54..efea3bd5bc 100644
  
  	@memoize
  	get settingsResource(): URI { return joinPath(this.userRoamingDataHome, 'settings.json'); }
+@@ -284,7 +294,12 @@ export class BrowserWorkbenchEnvironmentService implements IWorkbenchEnvironment
+ 						extensionHostDebugEnvironment.params.port = parseInt(value);
+ 						break;
+ 					case 'enableProposedApi':
+-						extensionHostDebugEnvironment.extensionEnabledProposedApi = [];
++						try {
++							extensionHostDebugEnvironment.extensionEnabledProposedApi = JSON.parse(value);
```

Comment:
wtf it only lets you pass in a string?

Reply:
lol yeah I think it's technically possible to put other things in there but that's what the types say.